### PR TITLE
398517 - reduce cognitive complexity

### PR DESCRIPTION
**Pull request by Afshin-Karamifar-Capgemini**
## Checklist

- [ ] Content has no typos
- [ ] Correct inline validation errors
- [ ] Back link and next page navigation
- [ ] Removed all unnecessary console logs
- [ ] Removed all commented out code
- [ ] Updated README / confluence

### CODE REVIEW

File: test/utils/check-answers.js [hunk 1]
```diff
@@ -474,6 +474,7 @@ module.exports = {
   expectedEndemicsFollowUpDairy,
   expectedEndemicsFollowUpPigs,
   expectedEndemicsFollowUpSheep,
+  sheepTestResults,
```
Comment (alex742) [inline test/utils/check-answers.js:477]: Why do we need to export this? It isn't used anywhere as far as I can see

Comment (Afshin-Karamifar-Capgemini) [reply test/utils/check-answers.js:477]: It's been used in check-answers.test.js

Comment (alex742) [reply test/utils/check-answers.js:477]: Do we need to export it just to test it, is it being flagged as an uncovered line?

Comment (Afshin-Karamifar-Capgemini) [reply test/utils/check-answers.js:477]: Yes, we need it to cover line 286 to 290

Comment (alex742) [reply test/utils/check-answers.js:477]: Is it used anywhere, if it isn't being covered by tests then that implies that it isn't actually being used. Or can we test wherever it is used instead?

**Pull request merged**